cs/tech.go

```diff
@@ -119,14 +121,97 @@ type TechHullComponent struct {
 	Power                     int           `json:"power,omitempty"`
 	Range                     int           `json:"range,omitempty"`
 	Initiative                int           `json:"initiative,omitempty"`
-	Gattling                  bool          `json:"gattling,omitempty"`
+	Gatling                   bool          `json:"gatling,omitempty"`
 	HitsAllTargets            bool          `json:"hitsAllTargets,omitempty"`
 	DamageShieldsOnly         bool          `json:"damageShieldsOnly,omitempty"`
 	Accuracy                  int           `json:"accuracy,omitempty"`
 	CapitalShipMissile        bool          `json:"capitalShipMissile,omitempty"`
 	CanJump                   bool          `json:"canJump,omitempty"`
 }
 
+// get actual armor/shield value for a tech item given its shield/armor amounts and the multipliers for each
```

Comment:
I set this to range ^2 because it was the lowest value I could think of that made range 3 beams beat out range 1 beams from the next tier while still being barely worse than range 2 beams.